docs/examples/tutorial/pure/c_compare.py

```diff
@@ -0,0 +1,5 @@
+@cython.cfunc
+@cython.returns(cython.bint)
+@cython.locals(a=cython.int, b=cython.int)
+def c_compare(a, b):

```

Comment:
We should generally avoid `@cython.locals()` and `@cython.returns()` and use type annotations instead.
```suggestion
@cython.cfunc
def c_compare(a: cython.int, b: cython.int) -> cython.bint:
```